clock/src/main/java/io/deephaven/clock/RealTimeClockLoader.java

```diff
@@ -0,0 +1,53 @@
+package io.deephaven.clock;
+
+import java.util.Iterator;
+import java.util.ServiceLoader;
+
+/**
+ * Created by rbasralian on 10/13/22
+ */
+class RealTimeClockLoader {
+
+    private static final String REAL_TIME_CLOCK_SOURCE_PROPERTY = "RealTimeClockLoader.timeSource";
+
+    // TODO: this should probably default to 'instant' and get overrided wherever we do add-exports.
+    private static final String REAL_TIME_CLOCK_SOURCE =
+            System.getProperty(REAL_TIME_CLOCK_SOURCE_PROPERTY, "instant").toLowerCase();
+
+    private static final String JDK_INTERNALS_CLOCK_CLASS_NAME = "io.deephaven.clock.impl.JdkInternalsRealTimeClock";
+
+    static RealTimeClock loadImpl() {
+        final RealTimeClock impl;
+        switch (REAL_TIME_CLOCK_SOURCE) {
+            case "jdk-internals":
+                final Iterator<RealTimeClock> it = ServiceLoader.load(RealTimeClock.class).iterator();
```

Comment:
Can you safely try jdk-internals if you have something like "default"; then degrade to instant?